storage/migrations/0009_oasis_3_emerald_init.up.sql

```diff
@@ -2,6 +2,9 @@
 
 BEGIN;
 
+-- Create schema for non-Damask-specific data, namely address preimages, which are compatible back to oasis-1.
```

Comment:
Thought experiment: When oasis_4 comes around, we'll presumably migrate all indexed data to oasis_4-compatible tables, then drop/archive oasis_3 tables. So I don't think it makes much sense to have a separate oasis_1 schema for things that are useful in oasis_3 but happen to be compatible with oasis_1.

If we really do want a separate schema, you also need to [update ops to grant permissions to all the relevant users](https://github.com/oasislabs/private-ops/blob/c4ca72e4c60d5a8b601c4c73912da5dbca0e3b72/infrastructure/gcp/production/us-west2/oasis-indexer-db/terragrunt.hcl#L46-L46) for future DB instances, and run `GRANT` (or terragrunt apply) manually for current DB instances.

Reply:
that wasn't my interpretation of how we'd manage the schemas. let's talk about this on slack

Reply:
ok moving this to oasis_3 with a comment not to erase it across hard forks

Reply:
... although we haven't agreed on what will happen on hard fork. this save us the trouble of adding new permissions. and entering stuff in the hard fork when we implement it rather than the farthest back compatible hard fork is also fine

Reply:
moved